### DISCUSSION TITLE
fastf1.get_event_schedule(2022) to image

### DISCUSSION THREAD

**Discussion by walteij** [theOehrly/Fast-F1#128] [Q&A]
Hi All,

I want to export the data from fastf1.get_event_schedule(2022) to a png file. I can select the columns (Country, EventDate, EventFormat etc) separately, but is there a way to either put this data in a dataframe or place it in a transparant png in a quick and easy way?

I'm fairly new to both python and FastF1, but I am really trying to get into this stuff and I am trying to figure out what the <class 'fastf1.events.EventSchedule'> comprehends.

OR... I could just do a `ff1.get_event_schedule(2022).to_dict()` and convert that to a dataframe to work with.... So I think I just answered my own question.

**Comment by theOehrly** [comment]
The EventSchedule is (a subclass of) a dataframe. So there's probably no need to convert it to one first?

And your problem then more generally is, how to render an image from a dataframe. But I can't really help you there, as I've never done that.

**Reply by walteij** [reply]
Thanks. On another note, in the eventschedule, what is the timezone that is used for the sessions?
It looks like the local times are being used. SO Melbourne is 2022-04-10 15:00:00 local time race, but my time (CEST) it is 07:00
So if I wanted to use the eventschedule to create some sort of calendar, it's not so easy, as the times are all in different timezones.
Is it possible to get all the times in GMT (of even better in UTC)?
